cfme/utils/template/openstack.py

```diff
@@ -9,7 +10,44 @@ class OpenstackTemplateUpload(ProviderTemplateUpload):
     log_name = 'RHOS'
     image_pattern = re.compile(r'<a href="?\'?([^"\']*(?:rhos|openstack|rhelosp)[^"\'>]*)')
 
+    @log_wrap('Deploy template to vm - before templatizing')
+    def deploy_vm_from_template(self):
+        """Deploy a instance from the raw template"""
+        self.mgmt.get_template(self.image_name).deploy(
+            vm_name=self.template_name,
```

Comment:
If this uses temp_template_name, then its consistent with the rhv process, and your property for fetching ssh credentials can be more consistent if its trying to get the VM after its been deployed here.